### PR TITLE
Include scripts in MyDumper docker image

### PR DESCRIPTION
**Pull request by kornesh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,5 +17,8 @@ RUN apk add --no-cache --update $LIB_PACKAGES $BUILD_PACKAGES \
   && (rm -rf /tmp/* 2>/dev/null || true) \
   && (rm -rf /var/cache/apk/* 2>/dev/null || true)
 
+COPY ./scripts /scripts
```
Comment (faizhasim) [inline Dockerfile]:
```
COPY ./scripts /usr/local/bin/mydumperscripts
ENV PATH "$PATH:/usr/local/bin/mydumperscripts"
```

no need to change `WORKDIR`.

Comment (kornesh) [reply Dockerfile]: Is there any specific reason for not changing the `WORKDIR`?

Comment (faizhasim) [reply Dockerfile]: Seems like those scripts are a bunch of cli utilities. `/usr/local/bin` is usually meant for normal users binaries. They're already part of the path, so no need to change `WORKDIR`. Just call like normal cli.

Plus, you can invoke any of the scripts with `docker run` directly. Like `docker run --rm sr/image-name dump-to-s3 ...`.

Also, if my assumption of your intention is correct, I think your `CMD` below can be removed.

Up to you boss, just menyibuk saja. hahah

Comment (kornesh) [reply Dockerfile]: Sorry, I thought I've already replied your comment. It makes sense, with your changes even if we `cd` to other directories the utility scripts would still work.

**Pull request merged**